_includes/podcast-links.html

```diff
@@ -0,0 +1,11 @@
+  <div id="podcast-links">
+    <a href="https://anchor.fm/s/6bf36dec/podcast/rss" title="Subscribe using RSS"><img src="/assets/images/podcast/rss.png" alt="RSS icon" /></a>
```

Comment:
This could be generated using a jekyll for loop. I don't think it's worth changing now, but if we find that we need to update these links occasionally, then it might be worth implementing to make this more maintainable.